packages/core/components/FileDetails/index.tsx

```diff
@@ -122,73 +75,25 @@ function resizeHandleDoubleClick() {
 /**
  * Right-hand sidebar of application. Displays details of selected file(s).
  */
-export default function FileDetails(props: FileDetails) {
-    const globalDispatch = useDispatch();
+export default function FileDetails(props: Props) {
     const [windowState, windowDispatch] = React.useReducer(windowStateReducer, INITIAL_STATE);
     const [fileDetails, isLoading] = useFileDetails();
-    const shouldDisplaySmallFont = useSelector(selection.selectors.getShouldDisplaySmallFont);
 
     // If FileDetails pane is minimized, set its width to the width of the WindowActionButtons. Else, let it be
     // defined by whatever the CSS determines (setting an inline style to undefined will prompt ReactDOM to not apply
     // it to the DOMElement altogether).
     const minimizedWidth =
         windowState.state === WindowState.MINIMIZED ? WINDOW_ACTION_BUTTON_WIDTH : undefined;
 
-    // If the file has a thumbnail image specified, we want to display the specified thumbnail. Otherwise, we want
-    // to display the file itself as the thumbnail if possible.
-    // If there is no thumbnail and the file cannot be displayed as the thumbnail- show a no image icon
-    let thumbnail = (
```

Comment:
This logic was consolidated between the `FileThumbnail` component and the `FileDetails.getPathToThumbnail` function